docs/getting-started.md

```diff
@@ -0,0 +1,144 @@
+- [Kadaster niet-natuurlijk persoon](#KadasterNietNatuurlijkPersoon)
+
+#### KadastraalOnroerendeZaak
+- Opvragen van 1 specifiek kadastraalonroerendezaak resource o.b.v. een kadastraalonroerendezaakidentificatie.
+- Opvragen van 1 specifiek kadastraalonroerendezaak resource o.b.v. een kadastraleaanduiding.
+- Opvragen van een collectie kadastraalonroerendezaak resources o.b.v. een adres (postcode, huisnummer, huisletter, huisnummertoevoeging).
+- Opvragen van een collectie kadastraalonroerendezaak resources, waarbij een kadaster natuurlijk persoon rechthebbende is, o.b.v. een kadasterpersoonidentificatie.
+- Opvragen van een collectie kadastraalonroerendezaak resources, waarbij een kadaster nietnatuurlijk persoon rechthebbende is, o.b.v. een kadasterpersoonidentificatie.
+- Opvragen van een collectie kadastraalonroerendezaak resources, waarbij een ingeschreven natuurlijk persoon rechthebbende is, o.b.v. een burgerservicenummer.
+
+#### ZakelijkGerechtigde
+- Opvragen van een collectie zakelijkgerechtigde resources van 1 specifiek kadastraalonroerendezaak o.b.v. een kadastraalonroerendezaakidentificatie.
+- Opvragen van 1 specifiek zakelijkgerechtigde resource o.b.v. een zakelijkgerechtigdeidentificatie.
+
+#### KadasterNatuurlijkPersoon
+- Opvragen van 1 specifiek kadaster natuurlijk persoon resource o.b.v. een kadasterpersoonidentificatie.
+- Opvragen van een collectie kadaster natuurlijkpersoon resources o.b.v. een zoekterm. (Zie feature [zoeken kadasternatuurlijkpersoon](../features/zoeken-kadasternatuurlijkpersoon.feature))
+
+#### KadasterNietNatuurlijkPersoon
+- Opvragen van 1 specifiek kadaster nietnatuurlijk persoon resource o.b.v. kadasterpersoonidentificatie.
+- Opvragen van een collectie kadaster nietnatuurlijk persoon resources o.b.v. een zoekterm. (Zie feature [zoeken kadasternietnatuurlijkpersoon](../features/zoeken-kadasternietnatuurlijkpersoon.feature))
+
+### Algemeen
+Verder zijn er nog een paar algemene functies die gelden voor alle bovenstaande aanvragen:
+- Gebruik van de fields parameter om de response te filteren. (Voor werking, zie feature [fields](https://github.com/VNG-Realisatie/Haal-Centraal-common/blob/v1.0.0/features/fields.feature))
+- Gebruik van de expand parameter om subresources te ontsluiten. (Voor werking, zie feature [expand](https://github.com/VNG-Realisatie/Haal-Centraal-common/blob/v1.0.0/features/expand.feature))
+- Velden die altijd worden geleverd. (Voor werking, zie feature [levering velden](../features/levering-velden.feature))
+
+|Resource                           |Velden                         |
+|-----                              |------                         |
+|KadastraalOnroerendeZaak           |identificatie<br/>type<br/>_links.self |
+|ZakelijkGerechtigde                |identificatie<br/>_links.self |
+|KadasterNatuurlijkPersoon          |identificatie<br/>_links.self |
+|KadasterNietNatuurlijkPersoon      |identificatie<br/>_links.self |
```

Comment:
De </br>  tags maken dat de velden ocver 3 regels verspreid worden. Dat was in de originele versie al zo, maar miesschien wel zo netjes om ze hier te verwijderen.

Reply:
ik heb de meegeleverde velden nu steeds op één regel gezet, gescheiden door komma's